flink-streaming-java/src/test/java/org/apache/flink/streaming/runtime/operators/windowing/CoGroupJoinITCase.java

```diff
@@ -234,6 +234,109 @@ public void invoke(String value) throws Exception {
 		Assert.assertEquals(expectedResult, testResults);
 	}
 
+
+	// TODO: design buffer join test
```

Comment:
As you can see in the code, the test case is implemented. I forgot to remove this comment.